packages/ui-react/src/components/privy-aa/provider.tsx

```diff
@@ -0,0 +1,100 @@
+import { PrivyClientConfig, PrivyProvider } from "@privy-io/react-auth";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import {
+	PimlicoClient,
+	createPimlicoClient,
+} from "permissionless/clients/pimlico";
+import { PropsWithChildren, createContext, useContext, useState } from "react";
+import { http } from "viem";
+import { entryPoint07Address } from "viem/account-abstraction";
+import { sepolia } from "viem/chains";
+import { WagmiProvider, createConfig } from "wagmi";
+
+interface PrivyAAContextProps {
+	appId: string;
+	pimlicoApiKey: string;
+}
+
+interface PrivyAAContext {
```

Comment:
thanks pls add loading pimlicoApiKey from process.env to domain/config.ts and updat env.sample